recipe/meta.yaml

```diff
@@ -33,7 +35,9 @@ requirements:
     - patch     # [not win]
     - m2-patch  # [win]
   host:
-    - openssl
+    - openssl {{ openssl }}
+  run:
+    - openssl 3.*
```

Comment:
```suggestion
    - openssl  # exact pin handled through openssl run_exports
```

Reply:
does this mean that I have to get rid of
`ignore_run_exports_from:
    - openssl`?

Reply:
Actually, OpenSSL isn't required at all. It's only required to run the tests (which we aren't running). You can remove OpenSSL.

Reply:
so.....I can basically abandon the whole PR since it's a rebuild? or should it at least be a dependency for the tests?

Reply:
DOn't abandon it! Remove the OpenSSL dependency because it's not used. We do not run the tests for this package right now.